### ISSUE TITLE
Issue converting gromacs topology files containing tip5p water models

### ISSUE THREAD

**Issue by NJPalmer15** [ParmEd/ParmEd#1107]
Hello, I am trying to convert a pair of Gromacs .gro/.top files into an Amber format. When loading the Gromacs files into parmed I run into this error:

parmed.exceptions.GromacsError: Only 3-point vsite type 1 is supported

After doing some digging into the parameter files that I've made I found that the vsites are in the tip5p model that I want to use in my simulations. Does this mean that I cannot convert a tip5p model from Gromacs to Amber? If not would there be a different way to convert between these file types?

My real issue is that I have an .itp file for a non-standard solvent molecule made for use with an Amber force field, and I want to have a mixed solvent system with tip5p and this non-standard solvent molecule.

Thank you in advance.

**Comment by swails** [comment]
That's correct, I haven't currently worked out the rules to translate other types of virtual sites to Amber.  I'm not aware of any other package that does GROMACS -> Amber file conversions, so I don't know that what you're trying to do is currently possible.
